app/models/profile_host.rb

```diff
@@ -8,4 +8,16 @@ class ProfileHost < ApplicationRecord
 
   validates :profile, presence: true
   validates :host, presence: true, uniqueness: { scope: :profile }
+
+  after_destroy :destroy_orphaned_host
+
+  def destroy_orphaned_host
+    return unless host.profiles.empty?
+
+    if Settings.async
+      DeleteHost.perform_async(id: host.id)
```

Comment:
Where is the change for `config/settings.yml` to add this setting?

Reply:
It was missing indeed and it was making the test for this fail, I just cp-ed a6150a2b8cc23f11c41357148c9b83e91ca196f9